### PR TITLE
Bug 1219618 - [System] RTL refacto for notifications.

### PR DESCRIPTION
**Pull request by autra**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=dea6bd109a5d39d4c7de6cf0ecaa2a1af5ae9480

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6b6e9a000d1e46a0b221e63c44a45625a4c47724

File: apps/system/js/notification_screen.js [hunk 1]
```diff
@@ -784,10 +774,12 @@ var NotificationScreen = {
     }).bind(this);
     lastClearable.addEventListener('transitionend', removeAll);
 
+    var disappearingClass = document.dir === 'ltr' ?
+      'disappearing-via-clear-all' : 'disappearing-via-clear-all-left';
```
Comment (etiennesegonzac) [inline apps/system/js/notification_screen.js]: :+1:

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d5ef4c7d11f806b3da241160c761dd3ff24e3e54

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=aa6874c79d45701c345bc7466fc941c4a99bb2b5

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=aa6874c79d45701c345bc7466fc941c4a99bb2b5

**Pull request merged**